config.sh

```diff
@@ -32,7 +32,7 @@ fi
 export TOOLCHAIN_NAME=rve-nightly
 
 # Artifact name should be versioned as it is only used at install time.
-export ARTIFACT_NAME=rust-rve-nightly-$rust_commit_date-$TOOLCHAIN_HOST_TRIPLET
+export ARTIFACT_NAME=rust-rve-nightly-1-$rust_commit_date-$TOOLCHAIN_HOST_TRIPLET
```

Comment:
It turns out that we don't need this. You can find the artifact if you drill down into the github actions. It just doesn't appear attached to the commit if it has the same name.